pkg/utils/proc/stat_test.go

```diff
@@ -45,14 +52,14 @@ func TestProcStatSize(t *testing.T) {
 	}
 }
 
-var content = "3529367 (Isolated (((Web))) Co) S 3437358 3433422 3433422 0 -1 4194560 " +
+var statContent = "3529367 (Isolated (((Web))) Co) S 3437358 3433422 3433422 0 -1 4194560 " +
 	"50679 0 0 0 566 643 0 0 20 0 29 0 46236871 2609160192 33222 " +
 	"18446744073709551615 94165013317536 94165014109840 140730010890672 " +
 	"0 0 0 0 16846850 1082134264 0 0 0 17 29 0 0 0 0 0 94165014122560 " +
 	"94165014122664 94165887094784 140730010895394 140730010895699 " +
-	"140730010895699 140730010898399 0\n"
+	"140730010895699 140730010898399 -1\n"
 
-func TestProcStatParsing(t *testing.T) {
+func Test_newProcStat(t *testing.T) {
```

Comment:
Wouldn't be good to add some more test cases?
With different `statContent` and maybe with different `statDefaultFields`?

Reply:
It would be nice having it for sure. Currently the stat (and status) is well coupled with its inner parts, so to provide different defaultFields and structs in a test, I believe that we would need to make use of 2 interfaces. I'm for maturing the idea, and if new fields are required we definitively should refactor it.